### DISCUSSION TITLE
Sharing a file by holding the file results into wrong URI for the file, GitHub mobile app on iPhone

### DISCUSSION THREAD

**Discussion by viveksinghggits** [community/community#130561] [Mobile] [answered]
### Select Topic Area

Bug

### Body

I am not really sure if it is just me or someone else also faced this issue on GitHub mobile app. When I navigate to particular source file in a repo and try to share it by touching it and holding to get the `Share via` option, I get the wrong URI of the file. Let's say I am looking at this dir https://github.com/kanisterio/kanister/tree/master/examples/mysql and when I share the file (let's say README.md) using the option `Share via` like mentioned above, the URI that I get is https://github.com/examples/mysql/README.md which is incorrect (doesnt' have org/repo name). But it works properly when I open the source file and then share it by clicking on the option `...` button on top right corner and then share it using `Share via...`.

**Comment by Iron-Ham** [comment, marked as answer]
Hi @viveksinghggits,

Thank you for reporting this issue! The team is aware of the bug and a fix will be released in version 1.168.0.


Thanks,
Hesham
